proposed/http-client/http-client.md

```diff
@@ -43,10 +43,9 @@ interface HttpClient
     /**
      * Sends a PSR-7 request.
      *
-     * If a request is sent without any prior configuration, an exception MUST NOT be thrown
-     * when a response is recieved, no matter the HTTP status code.
+     * An exception MUST NOT be thrown when a response is recieved, no matter the HTTP status code.
      *
-     * If a request is sent without any prior configuration, a HTTP client MUST NOT follow redirects.
+     * The HTTP client MUST NOT follow redirects.
```

Comment:
this is inconsistent with what we say above. and i would strongly opt to either MUST or MUST NOT redirects. 

while i prefer to not follow redirects in my applications, so that i can detect mistakes in my API clients (after all, if my client is redirected, it probably is redirected each time i do a request, which is very poor resource consumption) we should probably opt for the more forgiving choice of following redirects.

i feel that its fine to do allow configuration options to disable redirects in the client for when you want strict control (or in my case probably a plugin for the PluginClient to throw an exception when it encounters a 3xx state, so that i have to debug my problem)